TombEngine/Objects/TR5/Entity/tr5_guard.cpp

```diff
@@ -1532,4 +1534,22 @@ namespace TEN::Entities::Creatures::TR5
 			}
 		}
 	}
+
+	void GuardHit(ItemInfo& target, ItemInfo& source, std::optional<GameVector> pos, int damage, bool isExplosive, int jointIndex)
+	{
+		if (pos.has_value())
+		{
+			if (jointIndex == GUARD_HEAD_MESH)
```

Comment:
that is good for now I guess :) I'm a little bit hesitant because it's hardcoded and if people want to make a completely new guard then the head will have to be ID 14 but i think that's not a problem